### DISCUSSION TITLE
Pylontech Force H1/2/3 solarman stick ethernet/wireless ip address reset

### DISCUSSION THREAD

**Discussion by micronikje** [davidrapan/ha-solarman#110] [Q&A] [answered]
Hi,
Does somebody know how to reset the AP ip address of the solarman webpage? 
I added the app to homeassistant logged in and made a typing mistake in the AP ip address. saw it to late and now I can't get into the webpage again.

**Comment by PV-Joe** [comment]
> Hi,
> 
> Does somebody know how to reset the AP ip address of the solarman webpage? 
> 
> I added the app to homeassistant logged in and made a typing mistake in the AP ip address. saw it to late and now I can't get into the webpage again. 

What about resetting the LSW3 Stick-Logger and set it up again?

![image](https://github.com/user-attachments/assets/2af38f01-e41a-48ba-a013-a78c43d6a8e0)

**Comment by micronikje** [comment]
I don't use this logger. The Pylontech force H3 has a network port. I use a utp cable connected to my router. So maybe there is another method????

**Comment by gedger** [comment]
Can you clarify what the problem is, first you ask about resetting the AP ip address and then later say that it doesn't have an AP but a network port?

If you have changed the IP address on the Pylontech do you know what you have changed it too? If you do then plug a PC directly into the network port and then set it's IP address and subnet to be in range and then you should be able to make contact and reset the IP to the correct value.

If you don't know the IP address then you'd be better off finding a Pylontech forum and finding out how to reset the device to factory settings or reset the network settings. Have you tried Pylontech support directly?

**Comment by micronikje** [comment]
The situation is as follows: I have a solis inverter which is connected to the pylontech force h3 battery. The battery has a network port. This port is connected through a utp cable with my router. In homeassistant I have the solarman app. In a browser you can enter the wifi ip address and login to a webpage. In this webpage you can set the wifi ip address and a AP ip address. In this last ip address (AP) I made a mistake, which resulted in the fact that I now can't open the webpage and the solarman app in homeassistant does not work anymore. So I need a way to reset the mistyped AP ip address to let it all function again.

Thanks for your help.

Verzonden vanaf Outlook voor Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by PV-Joe** [reply]
Can't you see the new IP Adress in your router?

**Comment by micronikje** [comment]
yes I can. But it is not accessable.

Verzonden vanaf Outlook voor Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by gedger** [comment]
Can I clarify further please. 

So as far as I am aware Solis inverters do not have any network connectivity but instead have an RS485 Modbus port. Most installs have a Wi-Fi dongle or Ethernet dongle that is plugged into this RS485 port which then provides the Solarman interface that we all use with this integration 

So in your installation is the Solis RS485 port plugged into the Pylontech battery?

**Reply by micronikje** [reply]
Yes, that is right. But the Pylontech battery has a network port as I explained and is connected to my router. Through this network port I could access a webpage before I made the mistake.
See this explanation https://gathering.tweakers.net/forum/list_messages/2230992/184 in Dutch from cryptocake on how to access the nexus battery. the nexus battery is the pylontech battery. If needed I will try to translate his explanation on how to access the battery.

**Reply by gedger** [reply]
Thanks, auto translation worked fine. So it's a webpage being served by the Pylontech battery that you can no longer access, the Solis inverter has no role  in this case and was just confusing things. I would suggest finding out how to reset the Pylontech network settings from a dedicated Pylontech forum or even the manufacturer and then start again.

**Reply by PV-Joe** [reply]
So we are talking about how to access a Pylontech Battery here.

You might get better support on other platforms. This page is specialized on Solarman Stick Loggers.

@davidrapan You better change the topic as this has nothing to with Solarman and will be confusing for other users.

**Reply by gedger** [reply]
No it's OK as it seems that the Pylontech batteries do actually support the Solarman protocol. That's what threw me at the start. It kind of makes sense as the hybrid inverters now support batteries so why not have batteries support the same protocol and is supported by this integration. SO it's not just stick loggers anymore, some devices have it built it.

**Reply by davidrapan** [reply]
We in fact do support Pylontech Force batteries (They also use solarman protocol as @gedger mentioned). What I don't understand is what @micronikje misconfigured but maybe @cryptocake can help him?

**Reply by PV-Joe** [reply]
Sorry didn't know that Pylontech uses Solarman. Interesting though!

**Comment by gedger** [comment]
Not sure if anything in this document helps.

https://solar-distribution.baywa-re.de/out/media/05-Force-H2_Wi-Fi_Portal_Introduction_1.1_for_End-user.pdf

**Reply by micronikje** [reply]
I know this home.solarmanpv.com site and this site can actually access the battery. But I want the sensors shown in homeassistant through the solarman app.

**Reply by gedger** [reply]
The fact that the battery is still connected to the Internet and the Solarman service means that it has a valid IP address on your home network. So the router that hands out IP addresses should have a record of all the IP addresses on your network so it's a case of finding out which one is the battery. This can be a pain if a sensible network name hasn't been used by the manufacturer, sometimes it's a case of eliminating the known ones and see what's left.

**Reply by gedger** [reply]
Just realised that you stated that you do know the IP address but going to http://ipaddress:80 doesn't show the webpage anymore? I'm out of ideas....

**Reply by davidrapan** [reply]
Yeah well I would like to know where is the place he supposedly put the wrong ip?

**Reply by micronikje** [reply]
In the webpage there is a section called AP-ip address. There it went wrong. 
Now I'm going to try something with the battery. I will give my computer a ip address within the range of the original battery ip range (10.10.100.254) and connect my computer directly to the battery. Maybe that will help???

**Reply by gedger** [reply]
Me too, I've just been through my Solarman configuration screens and there's only a few points where WiFi is mentioned. First is the one to join a wireless dongle onto your local network. Obviously this makes no sense for an Ethernet connected device like the Pylontech. The second place is the Wireless Access point that is started when you first connect to the dongle to start the configuration process, again makes no sense for an Ethernet device. The third is the place where you can manually configure the devices IP address, gateway etc. It also has the WiFi details which I think is missing on an Ethernet dongle.

**Reply by gedger** [reply]
Posts crossed, does it look like this.
<img width="459" alt="Screenshot 2024-08-24 at 16 33 23" src="https://github.com/user-attachments/assets/cbdd90df-77fd-4e06-92cb-63809d1a383b">

**Reply by micronikje** [reply]
Yes, that is one of the webpages.

I tried as mentioned above to connect directly to the battery, but it didn't help.

**Reply by gedger** [reply]
So does a Pylontech battery has an access point as well as the Ethernet port? Is there a Wi-Fi antenna somewhere?

**Reply by micronikje** [reply]
Yes it also has a wifi antenna and I know the ip address. But sadly also not accessable.

**Reply by PV-Joe** [reply]
> Yes, that is one of the webpages.

Doesn't it have a reset button?
Maybe do a reset and use the default IP.

**Reply by micronikje** [reply]
The battery has a start button. I held it for 10 seconds to see if it would restart the system, but it did nothing.

**Comment by gedger** [comment]
So the page I showed is only for setting up the default WiFi which is broadcast when the device isn't configured properly. You haven't got an WiFi SSID being broadcast like AP_XXXXX? 

So did you use the wizard to join it to your home network or the Quick Set page?

**Reply by gedger** [reply]
Having just scanned 

https://en.pylontech.com.cn/view_pdf/web/viewer.html?file=/A_UpLoad/upload_file/20230901195746_0269.pdf&fname=RESS-Installation+Manual-Force%3fH3+Wi-Fi%3fPortal%3fIntroduction+（V1.0）5PMPA08-00229.pdf

It seems you use the network connector or the WiFi connection not both....

**Reply by micronikje** [reply]
No, no wifi ssid is being broadcasted. I used the wizard and it worked in homeassistant. Then I wanted to change the ip address and there it went wrong.

**Reply by gedger** [reply]
Do you know what you changed it to?

**Reply by PV-Joe** [reply]
Did you restart your router, just in case? Check IP after.

**Reply by micronikje** [reply]
The pdf revers to the home.solarmanpv.com site. On this site I can see my battery. I tried to use the windowws inspector on that site to see if I could find a ip address, but no luck.

**Reply by micronikje** [reply]
I know what I changed the ip address to and I restarted my router twice. But that ip address I changed it to does not show. But a new one that is not accessable.

**Reply by gedger** [reply]
So the IP address was still on the correct subnet for your network and it's not in use by anything else? If so it should show up. 

Can you power off/on the battery? 

Have you tried with the Ethernet cable disconnected? 

If not it keeps coming back to a factory reset however you do that....

**Reply by PV-Joe** [reply]
It needs a reset in my opinion.

Do you know how to reset it properly or did you just guess holding the reset button for 10 seconds?

**Reply by micronikje** [reply]
Thanks, I can power off/on the battery. But that is maybe a bit to risky. I think I will call the installer next week to find out if there is a easy reset possible.

**Reply by gedger** [reply]
Well report back what happens.

**Reply by cryptocake** [reply]
Somehow you can’t have the LAN and WiFi connected at the same time. If you have set the WiFi then be sure to disconnect the LAN cable. And also be sure to use an antenna (SMA male, 3dBi, 2.4GHz) in the antenna port (behind the red little cover).

**Reply by micronikje** [reply]
Thanks for the tip @cryptocake. 
I disconnected the lan cable and connected a wifi antenna and I can see now (very week signal) the AP_xxxx wifi ssid. But still can't connect to it. The standard password (admin) doesn't work. Or has it another standard password? If I can get in, maybe I can correct the settings then.

**Reply by gedger** [reply]
Double check your antenna has the correct connector, there are both reverse SMA and normal SMA. Make sure that one connector has a pin and the other connector has a socket. https://www.spo-comm.de/en/blog/know-how/what-is-the-difference-between-sma-and-rp-sma

The default username/password was admin/admin on my system.

**Reply by gedger** [reply]
Or try admin/123456789 as the username/password

**Reply by micronikje** [reply]
tried but didn't work.

**Reply by gedger** [reply]
I think some ruling came in that prohibits the use of default passwords so try the battery serial number otherwise no idea.

**Reply by cryptocake** [reply, marked as answer]
> Thanks for the tip @cryptocake. I disconnected the lan cable and connected a wifi antenna and I can see now (very week signal) the AP_xxxx wifi ssid. But still can't connect to it. The standard password (admin) doesn't work. Or has it another standard password? If I can get in, maybe I can correct the settings then.

You mean the SSID password? That is written on a sticker on the battery, next to the antenna ;-)

**Reply by micronikje** [reply]
@cryptocake Thanks that is the answer. I'm in and can reset the settings.
Everybody thanks for helping!!!!
